extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -43,6 +43,18 @@ amp-story-shopping-attachment a {
 }
 
 .i-amphtml-amp-story-shopping-plp-card {
+  /* Button resets. */
```

Comment:
Can we move this to its own class and share it in both places? i.e.
```html
<button class="i-amphtml-story-button-reset i-amphtml-amp-story-shopping-tag-inner" />
<button class="i-amphtml-story-button-reset i-amphtml-amp-story-shopping-plp-card" />
```

Reply:
Good call, it's a bit complicated because of how most of our code is in different shadow roots, so up to Philip to see how this could work and whether it's helpful

Reply:
Maybe we could do something like this? Happy to defer to Philip here to determine if its worth it
```js
import {CSS as attachmentCSS} from '...attachment.css';
import {CSS as sharedCSS} from '...shared.css';
...
createShadowRootWithStyle(container, element, attachmentCSS + sharedCSS)
```

...

Reply:
nice you did exactly this while I was typing the comment :) 
🎉🎉🎉🎉

Reply:
lol I left that last comment without refreshing and seeing your suggestion 😂